aten/src/ATen/native/quantized/cpu/qconv_unpack.cpp

```diff
@@ -230,14 +248,30 @@ TORCH_LIBRARY_IMPL(quantized, CatchAll, m) {
   m.impl("conv1d_unpack", TORCH_FN(QConv1dUnpackWeightsInt8::run));
   m.impl("conv2d_unpack", TORCH_FN(QConvUnpackWeightsInt8<2>::run));
   m.impl("conv3d_unpack", TORCH_FN(QConvUnpackWeightsInt8<3>::run));
+
   m.impl("conv2d_stride", TORCH_FN(QConvStride<2>::run));
   m.impl("conv2d_padding", TORCH_FN(QConvPadding<2>::run));
+  m.impl("conv2d_output_padding", TORCH_FN(QConvOutputPadding<2>::run));
   m.impl("conv2d_dilation", TORCH_FN(QConvDilation<2>::run));
   m.impl("conv2d_groups", TORCH_FN(QConvGroups<2>::run));
+  m.impl("conv2d_transpose", TORCH_FN(QConvTranspose<2>::run));
+
   m.impl("conv3d_stride", TORCH_FN(QConvStride<3>::run));
   m.impl("conv3d_padding", TORCH_FN(QConvPadding<3>::run));
+  m.impl("conv3d_output_padding", TORCH_FN(QConvOutputPadding<3>::run));
   m.impl("conv3d_dilation", TORCH_FN(QConvDilation<3>::run));
   m.impl("conv3d_groups", TORCH_FN(QConvGroups<3>::run));
+  m.impl("conv3d_transpose", TORCH_FN(QConvTranspose<3>::run));
+
+  // ConvTranspose is the same, however, we want to have different name.
+  m.impl("conv_transpose2d_unpack", TORCH_FN(QConvUnpackWeightsInt8<2>::run));
+
+  m.impl("conv_transpose2d_stride", TORCH_FN(QConvStride<2>::run));
+  m.impl("conv_transpose2d_padding", TORCH_FN(QConvPadding<2>::run));
+  m.impl("conv_transpose2d_output_padding", TORCH_FN(QConvOutputPadding<2>::run));
+  m.impl("conv_transpose2d_dilation", TORCH_FN(QConvDilation<2>::run));
+  m.impl("conv_transpose2d_groups", TORCH_FN(QConvGroups<2>::run));
+  m.impl("conv_transpose2d_transpose", TORCH_FN(QConvTranspose<2>::run));
```

Comment:
why do we need different names?

Reply:
You mean in addition to the `conv2d_*`? If so, this is just for consistency. They call the same functions, but because we have `conv2d` and `conv_transpose2d`, I thought it would be confusing to have `conv2d_*`, but not the `conv_transpose2d*`. LMK if I should change it

Reply:
maybe we can avoid the duplicate methods and add a comment?

Reply:
Same as my previous comment -- I am fine removing it, the only convern is the symmetry with respect to the conv_Xd methods.